tests/functional/smoke_tests/features/steps/dashboard_page.py

```diff
@@ -0,0 +1,28 @@
+from behave import then, when
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.wait import WebDriverWait
+
+
+@when('They click on an collection exercise period')
+def click_on_collex(context):
+    WebDriverWait(context.browser, 10).until(
+        EC.visibility_of_element_located((By.ID, 'collex-datatable')))
+    context.browser.find_elements_by_id('collex-datatable')[0].click()
+
+
+@then('They can view report figures on that collection exercise')
+def get_report_figures(context):
+    try:
+        assert context.browser.find_element_by_id('downloads-counter').text.isdigit()
```

Comment:
Once we display the survey type, ie eq/seft, we can prob check using that instead of assuming, if one isn't there the other would be.